types/actions.pp

```diff
@@ -1,31 +1,31 @@
 # Struct data type alias for Rsyslog Actions
 type Rsyslog::Actions = Array[
   Struct[{
-    name          => String[1],
-    type          => Rsyslog::Modules::Output,
-    facility      => Optional[String[1]],
-    action_params => Optional[Rsyslog::Actions::Parameters],
-    queue_params  => Optional[Rsyslog::Queue::Parameters],
-    output_params => Optional[Variant[
-      Rsyslog::Actions::Outputs::Omamqp1,
-      Rsyslog::Actions::Outputs::Omelasticsearch,
-      Rsyslog::Actions::Outputs::Omfile,
-      Rsyslog::Actions::Outputs::Omfwd,
-      Rsyslog::Actions::Outputs::Omhiredis,
-      Rsyslog::Actions::Outputs::Omhttpfs,
-      Rsyslog::Actions::Outputs::Omjournal,
-      Rsyslog::Actions::Outputs::Omkafka,
-      Rsyslog::Actions::Outputs::Omlibdbi,
-      Rsyslog::Actions::Outputs::Ommail,
-      Rsyslog::Actions::Outputs::Ommongodb,
-      Rsyslog::Actions::Outputs::Ommysql,
-      Rsyslog::Actions::Outputs::Ompgsql,
-      Rsyslog::Actions::Outputs::Ompipe,
-      Rsyslog::Actions::Outputs::Omprog,
-      Rsyslog::Actions::Outputs::Omrelp,
-      Rsyslog::Actions::Outputs::Omsnmp,
-      Rsyslog::Actions::Outputs::Omudpspoof,
-      Rsyslog::Actions::Outputs::Omusrmsg,
-    ]],
+      name          => String[1],
+      type          => Rsyslog::Modules::Output,
+      facility      => Optional[String[1]],
+      action_params => Optional[Rsyslog::Actions::Parameters],
+      queue_params  => Optional[Rsyslog::Queue::Parameters],
+      output_params => Optional[Variant[
+          Rsyslog::Actions::Outputs::Omamqp1,
+          Rsyslog::Actions::Outputs::Omelasticsearch,
```

Comment:
we're currently having a discussion on our IRC channel why some of us don't like this change, while others are welcoming it

until that discussion is over, these changes should not be merged

Reply:
@igalic Any idea how those discussions went?

Reply:
Would be great if the discussion took place over the mailing list so asynchronous feedback could be given. If a project maintainer like @dhollinger does not know how the conversation went, that is not a good sign of transparency.